code/online_challenges/src/hackerrank/3D_aurface_area/3D_surface_area.cpp

```diff
@@ -0,0 +1,52 @@
+#include < iostream >
+
+int main()
+{
+    int h, w;
+    int a[110][110];
```

Comment:
Use a vector so that an arbitrary size can be specified.

Reply:
Sir, no need of using vectors, it's under a limit which is given in the question.